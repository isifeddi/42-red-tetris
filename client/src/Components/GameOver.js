import React from "react";
import url from "../img/tenor.gif";
import url1 from "../img/README.gif";
import { StyledOverlay, StyledOverlayText } from "./styling/StyledStage";

export default function GameOver({ player }) {
  return (
    <StyledOverlay>
      <StyledOverlayText>
        <h1>{player?.lost === true ? "You lost" : "You Won"}</h1>
        {player?.lost === true ? (
          <img src={url} />
        ) : (
          <img style={{ width: "50%" }} src={url1} />
        )}
        <p>{player?.admin === true ? "Click on ‚ü≥ to restart the game" : "Wait for host player to restart the game"}</p>
      </StyledOverlayText>
    </StyledOverlay>
  );
}
