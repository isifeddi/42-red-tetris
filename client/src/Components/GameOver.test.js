import renderer from "react-test-renderer";
import GameOver from "./GameOver";

test("App render test ", () => {
  const tree = renderer.create(<GameOver />).toJSON();
  expect(tree).toMatchSnapshot();
});
