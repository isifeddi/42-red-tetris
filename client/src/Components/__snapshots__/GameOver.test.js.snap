// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render test  1`] = `
<div
  className="sc-gsTCUz kMFVFf"
>
  <div
    className="sc-dlfnbm gfmIzg"
  >
    <img
      src="README.gif"
      style={
        Object {
          "width": "200px",
        }
      }
    />
    <h1>
      You Won
    </h1>
    <p>
      Wait for host player to restart the game
    </p>
  </div>
</div>
`;
